package.json

```diff
@@ -35,6 +35,8 @@
     "fuse.js": "^3.3.0",
     "graphiql": "~0.13.0",
     "graphql": "^14.1.1",
+    "graphql-language-service-interface": "2.8.2",
+    "graphql-language-service-parser": "1.9.0",
```

Comment:
have to pin this now because there is a regression in latest releases of these and cause webpack to compain to having suitable loaders

see issue 1832 in the `graphiql` repo